bitnami/contour/values.yaml

```
@@ -840,6 +840,10 @@ envoy:
     ## @param envoy.service.ipFamilyPolicy [string], support SingleStack, PreferDualStack and RequireDualStack
     ##
     ipFamilyPolicy: ""
+    ## @param envoy.service.ipFamilies [array] List of IP families (e.g. IPv4, IPv6) assigned to the service.
+    ## Ref: https://kubernetes.io/docs/concepts/services-networking/dual-stack/
+    ipFamilies: []
+    ##  - IPv6
```

Just a minor comment, we tend to set the examples above the parameters. Does it make sense for you as well?

```suggestion
    ## E.g.
    ## ipFamilies:
    ##   - IPv6
    ##
    ipFamilies: []
```

---

hi, fixed